src/main/java/core/basesyntax/dao/FruitStorageDao.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.FruitStorage;
+
+public class FruitStorageDao {
+    public void create(String fruitName, int quantity) {
+        FruitStorage.storage.put(fruitName, quantity);
+    }
+
+    public void add(String fruitName, int quantity) {
+        paramsValidator(fruitName, quantity);
```

Comment:
```suggestion
        validateParams(fruitName, quantity);
```
use verbs for method names